straxen/contexts.py

```diff
@@ -22,7 +22,7 @@
     n_nveto_pmts=120,
     n_tpc_pmts=straxen.n_tpc_pmts,
     n_top_pmts=straxen.n_top_pmts,
-    gain_model=('to_pe_constant', 'TemporaryGXe_1500V_PMT116_1300_PMT195_1300'),
+    gain_model=('to_pe_constant', 'TemporaryGXe_1500V_PMT116_1300_PMT195_1300', 'TemporaryLXe_2e6HVmap_cutoff_1310'),
```

Comment:
Sadly it does not work likt this, you can specify just one in this 'to_pe_contstant' setting. There are other options but I don't think it makes too much sense to exploit them as we hopefully have a CMT soon.